types/react/test/next.tsx

```diff
@@ -2,6 +2,19 @@
 
 import React = require('react');
 
+interface Source {
+    value: string;
+}
+
+const source: Source = { value: '' };
+
+const mutableSource = React.unstable_createMutableSource(source, () => source.value);
+
+const getSnapshot = (source: Source) => source.value;
+
+// subscribe progress is ignored
+const subscribe: React.MutableSourceSubscribe<Source> = (source, callback) => () => {};
+
```

Comment:
Let's just use the [documented example](https://github.com/reactjs/rfcs/blob/master/text/0147-use-mutable-source.md#browser-apis) so that we have a real world use case. We wouldn't want people to copy this particular code.

Reply:
That's my opinion at the beginning, but there was an error: `TypeScript@4.4 compile error: Cannot find name 'Window'.`
Maybe there is a way to refer to it but I don't know?

Reply:
You would need to add `dom.d.ts` to the used libs in `types/react/tsconfig.json`:
```diff
        "lib": [
            "es6",
+           "dom",
        ],
```
Hopefully this doesn't affect publishing.

Reply:
It works on this test case, but some error occurs in `react/tests/index.js`.
I will take a look.👀

Reply:
I've fixed the error occurs in `react/tests/index.js`, but frankly I'm not sure at all.

Reply:
I'm offering that we implement [another documented example](https://github.com/reactjs/rfcs/blob/master/text/0147-use-mutable-source.md#selectors-that-use-props) which will not use dom API.
I will write a codesandbox to ensure it is well-practiced code.
And revert the tests to keep as it is.